### DISCUSSION TITLE
Change the file path that Dendron installs to

### DISCUSSION THREAD

**Discussion by n-zuk** [dendronhq/dendron#3597] [Q&A]
Is there a way to change the file path that Dendron installs to? I'd like to change it to an external drive rather than my local User folder.

**Comment by Yohn** [comment]
I'm also looking for this. I found the setting to change the root path within the settings, but I would still like to move all of the installed files to my D drive instead of the C drive. 
Reasoning -- I have 1TB of space on my D drive, and 128GB on my C drive. The C drive is practically out of space already. Lately, if I cant install and use a program on my D drive I end up not using it.

VSCode is installed via the portable mode, which allows me to put it on my D drive. It would be awesome if Dendron could use the data folder within the VSCode setup

**Comment by ScriptAutomate** [comment]
## Vault Locations

Hey! Have you tried doing:

- Open command palette (`Ctrl+Shift+P` on Windows/Linux, `Cmd+Shift+P` on MacOS)
- `Dendron: Initialize Workspace`
- Select location

This should bring up a file/directory navigator for you to get your vault created in a location of choice, including an external hard drive. Would that be what you are looking for?

> **Note**
> I think it may be `Dendron: Launch Tutorial` that installs to a default location, and adds in Tutorial information. Where this gets created, the directory can still be moved to a new location and can be opened/used as normal.

## Extension Modifications

May be of help if the question is about VS Code / VSCodium extensions:

- https://stackoverflow.com/questions/40080793/is-there-a-way-to-change-the-extensions-folder-location-for-visual-studio-code
- https://code.visualstudio.com/docs/editor/portable
- https://github.com/VSCodium/vscodium/blob/master/DOCS.md#portable
- https://code.visualstudio.com/docs/editor/extension-marketplace#_configuring-extensions
